clients/fides-js/docs/interfaces/FidesOptions.md

```diff
@@ -81,6 +81,10 @@ To use the `fides_embed` option, ensure that a DOM element with
 `id="fides-embed-container"` exists on the page, which FidesJS will then
 use as the parent element to render within.
 
+Before version 2.34 only the consent modal was embedded while the banner was
+not shown at all, but since that version both will be embedded. To get the
+previous behavior also set `fides_disable_banner` to `true`.
```

Comment:
nit: I'd rather not reference "previous versions" here, just state the fact:
* will embed banner & modal
* if you don't want banner, disable it

does that make sense?